lib/simple_enum/translation.rb

```diff
@@ -15,7 +15,7 @@ def human_enum_name(enum, key, options = {})
       defaults << key.to_s.humanize
 
       options.reverse_merge! count: 1, default: defaults
-      I18n.translate(defaults.shift, options)
+      I18n.translate(defaults.shift, **options)
```

Comment:
i am not a maintainer, or contributor to this project, but seems brittle to me to have this whole file duplicated to fix a single line.

We could use rescue here the exception so we avoid duplicating unnecesary code. Just my thoughts